schemaregistry/schemaregistry.md

```diff
@@ -405,3 +497,196 @@ The returned ´Content-Type´ is the same that was passed when the schema versio
+near the consumers.
+
+These two models mights also be mixed. Schemas representing event data that is
+used throughout a greater system might stem from a centrally managed registry
+and event data schemas that are local to a subsystem might be controlled by the
+respective producers. 
+
+### 4.2 Replication topologies
+
+A peer-to-peer replication topology copies schemas between a source and target
+registry directly. For a mutual synchronization of schemas between registries, a
+second replication path is established with reversed roles.
+
+A hub replication topology copies schemas from a source to the target through an
+intermediary hub. The hub acts as an intermediary store, but with the authority
+over the schemas still being with the producers. Hubs might also be used by
+services that cannot host their own service registry endpoints.
+
+A star replication topology uses hubs central hubs into which all sources
+contribute schemas and to which all targets subscribe. If the hub is a centrally
+governed registry, it might not accept inbound replications.
+
+### 4.3 Push replication
+
+In an imperative "push" replication model, a registry might push any updates of
+its state to one of more configured target registry by forwarding all changes to
+the respective schema registry API calls on that target registry.
+
+The pushing registry might be configured to copy changes of the full registry, of
+a specific schema group, or just of a specific schema.
+
+This replication model uses the registry API as defined above on the target
+registry and the configuration of what aspects of the registry are pushed are
+implementation specific.
+
+### 4.4 Pull replication
+
+In an imperative "pull" replication model, a target registry will once or periodically
+traverse the source registry contents and apply differences between the source
+state and its own state.
+
+The pulling registry might be configured to copy changes of the full registry, of
+a specific schema group, or just of a specific schema.
+
+This replication model uses the registry API as defined above on the source
+registry and the configuration of what aspects of the registry are pulled are
+implementation specific.
+
+### 4.5. Event-driven replication
+
+Event-driven replication uses CloudEvents to notify interested parties of
+changes in the source registry such that those parties can
+[pull](#44-pull-replication) changes immediately and as they become available.
+
+Each registry SHOULD offer a [Subscription API](../subscriptions-api.md)
+endpoint, either directly or using some middleware, to allow interested parties
+to subscribe to these change events.
+
+The subscription's `source` MUST be the root of the schema registry. A [prefix
+filter](../subscriptions-api.md#prefix-filter-dialect) on the `subject`
+attribute MAY be used to scope the subscription to a particular schema group or
+schema.
+
+The subscribing party registers a notification sink of its choosing.
+
```

Comment:
The models all seem to imply a snapshot or ETL type of interaction. A history is not offered nor is a request for "all changes since offset/epoch/timestamp X". Is that intentional?